### PR TITLE
docs: improved the en version of online_request spec

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
changed `online serving` to `online request`
fixed some links
improved the format of example

### CODE REVIEW

File: docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md [hunk 1]
```diff
@@ -1,72 +1,76 @@
-# SQL On-Line Specifications and Requirements
+# Online Specifications and Requirements for SQL
 
-OpenMLDB Online Serving provides real-time feature extraction services. The [DEPLOY](../deployment_manage/DEPLOY_STATEMENT.md) command of OpenMLDB deploys a piece of SQL text to the wire. After the deployment is successful, users can perform feature extraction calculations on the request samples in real time through the Restful API or JDBC API. Not all SQL can be deployed to provide services online. OpenMLDB has a set of specifications for online statements and OP.
+OpenMLDB can provide real-time feature extraction services under *online request* mode. The [DEPLOY](../deployment_manage/DEPLOY_STATEMENT.md) command can deploy a piece of SQL text online. If the deployment is successful, users can perform feature extraction on the requested samples in real time through the Restful API or JDBC API. Note that only some SQL commands can be deployed to provide services online. To deploy these SQL commands please follow the specifications below.
```
Comment (lumianph) [inline docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md:3]: a piece of SQL text
a SQL script

Comment (lumianph) [inline docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: a piece of SQL text 
=>
a SQL script

Comment (lumianph) [reply docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: perform feature extraction on the requested samples in real time
perform real-time feature extraction

File: docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md [hunk 2]
```diff
@@ -1,72 +1,76 @@
...
 -- desc: SELECT all columns
 SELECT * FROM t1;
   
--- desc: SELECT expression renamed
+-- desc: rename expression 1
 SELECT COL1 as c1 FROM t1;
  
--- desc: SELECT expression rename 2
+-- desc: rename expression 2
 SELECT COL1 c1 FROM t1;
```
Comment (lumianph) [inline docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: is this example correct?

File: docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md [hunk 3]
```diff
@@ -86,27 +90,12 @@ desc t1;
...
-### Window Usage Specification Under Online Serving
+### Specifications of WINDOW under Online Request Mode
 
-- Window borders only support `PRECEDING` and `CURRENT ROW`
-- Window types only support `ROWS` and `ROWS_RANGE`
-- The window `PARTITION BY` can only support column expressions, and the column needs to hit the index
-- The window `ORDER BY` can only support column expressions, and the column needs to hit the time column of the index
+- Window boundary: only `PRECEDING` and `CURRENT ROW` are supported.
+- Window type: only `ROWS` and `ROWS_RANGE` are supported.
+- `PARTITION BY` only supports column expressions, and the column needs to hit the index.
```
Comment (lumianph) [inline docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: needs to be indexed as a `KEY`.

File: docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md [hunk 4]
```diff
@@ -86,27 +90,12 @@ desc t1;
...
+- `ORDER BY` only support column expressions, and the column needs to hit the time column of the index.
```
Comment (lumianph) [inline docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: needs to be indexed as a timestamp (`TS`).

File: docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md [hunk 5]
```diff
@@ -86,27 +90,12 @@ desc t1;
...
+- Another supported restrictions: `EXCLUDE CURRENT_ROW`, `EXCLUDE CURRENT_TIME`, `MAXSIZE` and `INSTANCE_NOT_IN_WINDOW`. See [WindowSpec elements specifically designed by OpenMLDB](../dql/WINDOW_CLAUSE.md#windowspec-elements-specifically-designed-by-openmldb) for detail.
```
Comment (lumianph) [inline docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: Other supported keywords:

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md (hunk 2 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: yes, it works

File: docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md [hunk 6]
```diff
@@ -1,72 +1,76 @@
-# SQL On-Line Specifications and Requirements
+# Online Specifications and Requirements for SQL
```
Comment (vagetablechicken) [inline docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: How about the file name? still serving

**Review by vagetablechicken — CHANGES_REQUESTED** [review]

File: docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md (hunk 6 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md]: the file name, corresponding index.rst files and links have been changed.
thx for reminding me~

**Review by vagetablechicken — APPROVED** [review]

**Review by lumianph — APPROVED** [review]

**Comment by lumianph** [comment]
@michelle-qinqin please resolve the conflicts

**Pull request merged**